packages/flutter/lib/src/rendering/sliver.dart

```diff
@@ -624,7 +676,7 @@ abstract class RenderSliver extends RenderObject {
   @override
   SliverConstraints get constraints => super.constraints;
 
-  // layout output
+  /// The amount of space this sliver occupies.
```

Comment:
...should only be set by `performLayout`.
...only valid after layout is complete.